site/en/blog/media-updates-in-chrome-58/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Media updates in Chrome 58
+description: >
+  Media controls customization, autoplay for Progressive Web Apps added to the
+  home screen, pause the autoplaying of muted video when invisible, and
+  color-gamut media query are there!
+layout: 'layouts/blog-post.njk'
+date: 2017-03-21
+updated: 2020-07-24
+authors:
+  - beaufortfrancois
+tags:
+  - media
```

Comment:
Can we add `- chrome-58` and similar tags to all these posts? I know it might be a pain, but we like to categorize stuff by release officially.

Reply:
Sure. See https://github.com/GoogleChrome/developer.chrome.com/pull/1365/commits/b523d5a4e3e500ad9d0c29797a5d82e22b8d79a6